components/application-mgt/org.wso2.carbon.identity.application.mgt.ui/src/main/java/org/wso2/carbon/identity/application/mgt/ui/ApplicationBean.java

```diff
@@ -829,6 +838,43 @@ public void setClaimUris(String[] claimUris) {
     }
 
     /**
+     * Check for custom authenticator
+     * @param authType authentication type
+     * @return boolean is custom authenticator
+     */
+    private boolean isCustomInboundAuthType(String authType) {
+        return !standardInboundAuthTypes.contains(authType);
+    }
+
+    /**
+     * Get all custom authenticators
+     * @return Custom authenticators
+     */
+    public List<InboundAuthenticationRequestConfig> getCustomInboundAuthenticators() {
```

Comment:
If we remove standard authenticators we have to rename this method as getInboundAuthenticators()